genai-perf/tests/test_openai_chat_converter.py

```diff
@@ -104,7 +130,7 @@ def test_convert_with_request_parameters(self):
             model_name=["test_model"],
             model_selection_strategy=ModelSelectionStrategy.ROUND_ROBIN,
             output_format=OutputFormat.OPENAI_CHAT_COMPLETIONS,
-            add_stream=True,  # set streaming
```

Comment:
Aw, but that "set streaming" comment really helped me understand what `add_stream=True` was doing 😩